cegs_portal/static/css/main.css

```diff
@@ -0,0 +1,46 @@
+.content-container {
```

Comment:
This file is included in base.html, but much of it is only used in experiment.html. The experiment.html classes could be included in the experiment.html <style> tag directly since there isn't much of it. 

Anything else that's needed globally (or nearly so) can be added to the project.css.tw file, where non-page-specific custom CSS goes. See: https://tailwindcss.com/docs/adding-custom-styles.